<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabMeasurementObject</class>
 <widget class="QWidget" name="TabMeasurementObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="boxProgress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>153</y>
     <width>319</width>
     <height>116</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Progress Monitor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>300</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Frame: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProgressIndex">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>25</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentMeasurement">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>45</y>
      <width>76</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>136</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Measurement:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameTimeResolved">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>8</y>
     <width>362</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>357</width>
      <height>264</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridTimeResolved">
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="comboPeriodUnit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;
Frame period between exposures. 
&lt;/nobr&gt;&lt;br&gt;&lt;nobr&gt;
 #period#
&lt;/nobr&gt;</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>hr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>us</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ns</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QComboBox" name="comboTimingMode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Timing Mode of the detector. 
 #timing#</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trigger Exposure Series</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Gated with fixed number</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Burst Trigger</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblNumTriggers">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Number of Triggers to be expected.
 #cycles#</string>
       </property>
       <property name="text">
        <string>Number of Triggers:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QSpinBox" name="spinNumTriggers">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Number of Triggers to be expected.
 #cycles#</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblDelay">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The Delay between Trigger Edge and Start of Exposure ( or Readout). 
#delay#</string>
       </property>
       <property name="text">
        <string>Delay After Trigger:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="spinDelay">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The Delay between Trigger Edge and Start of Exposure ( or Readout). 
#delay#</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>9</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2000000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QComboBox" name="comboDelayUnit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The Delay between Trigger Edge and Start of Exposure ( or Readout). 
#delay#</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>hr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>us</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ns</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblNumFrames">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Number of  Frames:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QSpinBox" name="spinNumFrames">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Number of measurements (not in real time) that will be acquired. 
 #frames#</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblExpTime">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;
Exposure Time of a frame.
&lt;/nobr&gt;&lt;br&gt;&lt;nobr&gt;
 #exptime#
&lt;/nobr&gt;</string>
       </property>
       <property name="text">
        <string>Exposure Time:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="spinExpTime">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;
Exposure Time of a frame.
&lt;/nobr&gt;&lt;br&gt;&lt;nobr&gt;
 #exptime#
&lt;/nobr&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>9</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2000000000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="comboExpUnit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;
Exposure Time of a frame.
&lt;/nobr&gt;&lt;br&gt;&lt;nobr&gt;
 #exptime#
&lt;/nobr&gt;</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>hr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>us</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ns</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblPeriod">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;
Frame period between exposures. 
&lt;/nobr&gt;&lt;br&gt;&lt;nobr&gt;
 #period#
&lt;/nobr&gt;</string>
       </property>
       <property name="text">
        <string>Acquisition Period:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="spinPeriod">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;nobr&gt;
Frame period between exposures. 
&lt;/nobr&gt;&lt;br&gt;&lt;nobr&gt;
 #period#
&lt;/nobr&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>9</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2000000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblTimingMode">
       <property name="text">
        <string>Timing Mode:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblNumSamples">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Number of Triggers to be expected.
 #cycles#</string>
       </property>
       <property name="text">
        <string>Number of Samples:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QSpinBox" name="spinNumSamples">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Number of Triggers to be expected.
 #cycles#</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>spinNumSamples</zorder>
  </widget>
  <widget class="QFrame" name="frameNotTimeResolved">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>336</width>
     <height>159</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>321</width>
      <height>130</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Number of Measurements:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="spinNumMeasurements">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Number of measurements (not in real time) that will be acquired. 
 #frames#</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="dispFileName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Root of the file name  - please check that the output directory is correctly set and select the file name format. 
 #fname#</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Run Index:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSpinBox" name="spinIndex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Run index (automatically incremented) 
 #index#</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>2000000000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkFile">
       <property name="toolTip">
        <string>&lt;nobr&gt;Sets output file name prefix&lt;/nobr&gt;&lt;br&gt;
&lt;nobr&gt;Check the box to enable write to file. &lt;/nobr&gt;</string>
       </property>
       <property name="text">
        <string>File Name:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>298</y>
     <width>318</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Starts Acquisition</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="icon">
       <iconset resource="../include/icons.qrc">
        <normaloff>:/icons/images/start.png</normaloff>:/icons/images/start.png</iconset>
      </property>
      <property name="shortcut">
       <string>Shift+Space</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnStop">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Stops Acquisition</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset resource="../include/icons.qrc">
        <normaloff>:/icons/images/stop.png</normaloff>:/icons/images/stop.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinNumMeasurements</tabstop>
  <tabstop>chkFile</tabstop>
  <tabstop>dispFileName</tabstop>
  <tabstop>spinIndex</tabstop>
  <tabstop>comboTimingMode</tabstop>
  <tabstop>spinNumFrames</tabstop>
  <tabstop>spinExpTime</tabstop>
  <tabstop>comboExpUnit</tabstop>
  <tabstop>spinPeriod</tabstop>
  <tabstop>comboPeriodUnit</tabstop>
  <tabstop>spinNumTriggers</tabstop>
  <tabstop>spinDelay</tabstop>
  <tabstop>comboDelayUnit</tabstop>
 </tabstops>
 <resources>
  <include location="../include/icons.qrc"/>
 </resources>
 <connections/>
</ui>
